examples/hello/bpReader/helloBPReader_nompi.cpp

```diff
@@ -34,7 +45,7 @@ int main(int argc, char *argv[])
 
         /** Engine derived class, spawned to start IO operations */
         adios2::Engine bpReader =
-            bpIO.Open("myVector_cpp.bp", adios2::Mode::Read);
+            bpIO.Open(filename.c_str(), adios2::Mode::Read);
```

Comment:
The `IO::Open` call takes an `std::string` for filename so no need for the `c_str`

Reply:
@chuckatkins : Yup, this is better. Fixed.